packages/meditrak-server/src/tests/routes/GETSurveyGroups/assertSurveyGroupsPermissions.test.js

```diff
@@ -0,0 +1,127 @@
+  const DEFAULT_POLICY = {
+    DL: ['Public'],
+    KI: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    SB: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Royal Australasian College of Surgeons'],
+    VU: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    LA: ['Admin'],
+  };
+
+  const models = getModels();
+  let surveyGroups = [];
+  let surveyGroup1;
+  let surveyGroup2;
+
+  before(async () => {
+    //Set up the survey groups and their surveys
+    surveyGroup1 = await findOrCreateDummyRecord(models.surveyGroup, {
+      id: 'TEST_SURVEY_GROUP_1',
+      name: `Test survey group 1`,
+    });
+    surveyGroup2 = await findOrCreateDummyRecord(models.surveyGroup, {
+      id: 'TEST_SURVEY_GROUP_2',
+      name: `Test survey group 2`,
+    });
+    surveyGroups = [surveyGroup1, surveyGroup2];
+
+    const adminPermissionGroup = await findOrCreateDummyRecord(models.permissionGroup, { name: 'Admin' });
+    const donorPermissionGroup = await findOrCreateDummyRecord(models.permissionGroup, { name: 'Donor' });
+    const vanuatuCountry = await findOrCreateDummyRecord(models.country, { code: 'VU' });
+    const laosCountry = await findOrCreateDummyRecord(models.country, { code: 'LA' });
+
+    await buildAndInsertSurveys(models, [
+      {
+        code: 'TEST_SURVEY_1',
+        name: 'Test Survey 1',
+        permission_group_id: adminPermissionGroup.id,
+        country_ids: [vanuatuCountry.id],
+        survey_group_id: 'TEST_SURVEY_GROUP_1',
+      },
+      {
+        code: 'TEST_SURVEY_2',
+        name: 'Test Survey 2',
+        permission_group_id: adminPermissionGroup.id,
+        country_ids: [vanuatuCountry.id],
+        survey_group_id: 'TEST_SURVEY_GROUP_1',
+      },
+      {
+        code: 'TEST_SURVEY_3',
+        name: 'Test Survey 3',
+        permission_group_id: donorPermissionGroup.id,
+        country_ids: [vanuatuCountry.id],
+        survey_group_id: 'TEST_SURVEY_GROUP_1',
+      },
+      {
+        code: 'TEST_SURVEY_4',
+        name: 'Test Survey 4',
+        permission_group_id: adminPermissionGroup.id,
+        country_ids: [laosCountry.id],
+        survey_group_id: 'TEST_SURVEY_GROUP_2',
+      },
+    ]);
+  });
+
+  describe('filterSurveyGroupsByPermissions()', async () => {
+    it('Sufficient permissions: Should filter any survey groups that users do not have access to any of the surveys within the group', async () => {
+      const accessPolicy = new AccessPolicy(DEFAULT_POLICY);
+      const result = await filterSurveyGroupsByPermissions(accessPolicy, models, surveyGroups);
+
+      expect(result.length).to.equal(2);
+      expect(result[0]).to.equal(surveyGroup1);
+      expect(result[1]).to.equal(surveyGroup2);
+    });
+
+    it('Insufficient permissions: Should filter any survey groups that users do not have access to any of the surveys within the group', async () => {
+      //Remove the permission of VU to have insufficient permissions to some surveys.
+      const policy = {
```

Comment:
```suggestion
      const { VU, ...policy } = DEFAULT_POLICY;
```
Can use this below as well

Reply:
I commented out only specific Admin permission instead of the whole line